docs/cow-protocol/tutorials/arbitrate/autopilot.md

```diff
@@ -1,6 +1,150 @@
+
+Much of the order data it needs is collected through the database, which is shared with the orderbook.
+The database stores the vast majority of available order information, including user signatures.
+
+Other information can only be retrieved onchain and is updated every time a new block is mined. For example, it needs to know from the protocol:
+
+- Which [pre-signatures](/cow-protocol/reference/core/signing-schemes#presign) have been set
+- If new [eth-flow orders](/cow-protocol/tutorials/cow-swap/native#eth-flow) have been created
+- Tracking which orders have been [invalidated](/cow-protocol/reference/contracts/core/settlement#invalidateorder) by the user
+- Detecting if a batch has been settled and it should prepare a new auction
+
+Retrieved information isn't limited to the CoW Protocol itself.
+The aution data also contains information on which liquidity is available to the solvers to settle an auction's orders.
+Solvers may use this liquidity information to help them out with finding a solution more quickly.
+Some liquidity sources are also used to provide a reference price of each token in an order;
+the reference price is used to normalize the value of the [surplus](/cow-protocol/reference/core/auctions/the-problem), since the surplus must be comparable for all orders and an order could use the most disparate ERC-20 tokens.
+
+Examples of liquidity sources that are tracked by the autopilot:
+- Uniswap (v2 and v3)
+- Balancer (Weighted, Weighted2Token, ComposableStable, ...)
+- SushiSwap
+- Swapr
+
+## Cutting auctions
+
+The autopilot constantly updates the list of orders that can be settled based on the data retrieved above.
+
+Orders that can't be settled are filtered out: these are expired orders, those with not enough balance, with missing approvals, or that use tokens that aren't supported by the protocol.
+
+The autopilot also checks that [ERC-1271](/cow-protocol/reference/core/signing-schemes#erc-1271) signatures are currently valid.
+
+This steps tries to guarantees that all available orders can be settled on CoW Protocol without further checks from the solvers.
+
+## Solver competition
+
+Once an auction is ready, the autopilot sends a `/solve` request to each solver's driver.
+Solvers have a short amount time (seconds) to come up with a [solution](/cow-protocol/core/auctions/the-problem#solution) and return its _score_ to the autopilot.
+The autopilot selects the winner based on this score once the allotted time expires or all drivers have returned their batch proposal.
+
+Up to this point, the autopilot only knows the score and not the full solution that achieves that score.
+The autopilot then asks the winning solver to reveal its score (throug `/reveal`) and then to execute the corresponding settlement transaction (`/settle`).
+The solver is responsible for executing the transaction onchain (through the driver).
```

Comment:
Making the differentiation of `driver` + `solver` is not needed from the perspective of the `autopilot`.